default/stringtables/de.txt

```diff
@@ -6300,7 +6300,7 @@ BLD_BLACK_HOLE_POW_GEN
 Schwarzes-Loch-Generator
 
 BLD_BLACK_HOLE_POW_GEN_DESC
-'''Erhöht auf allen via [[metertype METER_SUPPLY]] verbundenen Planeten mit Industriefokus [[metertype METER_TARGET_INDUSTRY]] um 1 per [[metertype METER_POPULATION]].
+'''Erhöht auf allen via [[metertype METER_SUPPLY]] verbundenen Planeten mit Industriefokus [[metertype METER_TARGET_INDUSTRY]] um 0.2 per [[metertype METER_POPULATION]].
```

Comment:
Looks like coma is used as decimal separator in that file. 

`0.2` -> `0,2`

Reply:
There were several points in which point was used, and I didn't care to check out what was the right form. It's indeed the comma. Fixed in all instances.